pkgs/games/crossfire/crossfire-server.nix

```diff
@@ -0,0 +1,34 @@
+{ pkgs, stdenv, fetchsvn, autoreconfHook, version, rev, sha256, maps, arch }:
+
+stdenv.mkDerivation rec {
+  version = "r${toString rev}";
+  name = "crossfire-server-${version}";
+
+  src = fetchsvn {
+    url = "http://svn.code.sf.net/p/crossfire/code/server/trunk/";
+    sha256 = sha256;
+    rev = rev;
+  };
+
+  nativeBuildInputs = with pkgs; [autoconf automake libtool flex perl check pkg-config];
```

Comment:
```suggestion
  nativeBuildInputs = [autoconf automake libtool flex perl check pkg-config];
```

Reply:
Breaks the build.

Reply:
You need to add all packages in the list to the header as well.